tests/wait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require(__DIR__ . '/../vendor/autoload.php');
+
+use Http\Client\HttpClient;
+use Http\Discovery\HttpClientDiscovery;
+use Http\Discovery\MessageFactoryDiscovery;
+use Http\Client\Exception\RequestException;
+
+$messageFactory = MessageFactoryDiscovery::find();
+$httpClient = HttpClientDiscovery::find();
+
+$route = 'http://127.0.0.1:8080/api/3.4/data.get';
+
+$body = '{"params": {"uuid": "cc1bbc0b-20e8-4e1f-b894-fb067e81c5dd"},"id":"request-3d254173"}';
```

Comment:
I cannot seem to find where this document gets populated in the k-search.. Is this part of some related tests?
Or is this expected to fail? (since you are just checking for 200 response code, which is also returned on error)

Nothing wrong perhaps, I'm just curious.

Reply:
That's a sort of an hack to check for the K-Search and K-Search engine being properly started. The call to `data.get` asks the engine (i.e. Solr) if a data exists. If Solr is not fully started the request fails with an empty page, while if succeed I get a 200 response containing an error object. In other words I'm interested only in understanding if Solr is properly started, but to do so I needed a valid request.